gui/gui.go

```diff
@@ -187,7 +187,8 @@ func layout(g *gocui.Gui, client *client.Client) error {
 			return err
 		}
 		addr := client.Address().Hex()[:10]
-		v.Title = client.Config.UserName + fmt.Sprintf(" %s on %s", addr, client.Config.ChainURL)
+		v.Title = client.Config.UserName + fmt.Sprintf(" %s on %s", addr,
+			client.Config.ChainURL(client.OpConfig.NetworkID))
```

Comment:
I think this expression is used several times and is worth being put into a method like `client.ChainURL() string`.